features/delete_zip_files/delete_zip_files.feature

```diff
@@ -1,8 +1,8 @@
-# @javascript
-# Feature:
-#   As a tester
-#   I want to be able to delete zip files on a ad hoc basis
-#   So that I can have confident in my automated test when the files gets to large to download
+@javascript
+Feature:
+  As a tester
+  I want to be able to delete zip files on a ad hoc basis
+  So that I can have confident in my automated test when the files gets to large to download
 
-#   Scenario: Delete all zip files
-#    When I have a list of zip files then I can delete them all
\ No newline at end of file
+  Scenario: Delete all zip files
+   When I have a list of zip files then I can delete them all
```

Comment:
Shouldn't this be - 
When I have a list of zip files
Then I can delete them all

Reply:
yeah but i don't want to have separate step because the there is only one line of code that does it all :)